### ISSUE TITLE
Label profiles

### ISSUE THREAD

**Issue by lonkelle** [lucaIz-ldx/MobileTSS#7]
I have two iPhone of the same types and your profiles look the same based on that. Could we perhaps, name these new profiles. 

Also, you added a lot in this update! Fantastic work. 🎉

**Comment by lucaIz-ldx** [comment]
I don’t see there is a point for that. No two devices can share ECIDs. You can already differentiate profiles from their ECIDs.

**Comment by lonkelle** [comment]
ECIDs are unique true, but impossible to remember which ECID belongs to which phone. You sure no naming of profiles is a possibility for this? I'll close the ticket if you don't want to enhance MobileTSS with this feature. :)

**Comment by lucaIz-ldx** [comment]
I am still not getting its purpose. Let's say you have 4 devices with same models. Given that you will save latest blobs for all of them, you just get blobs for that ECID when you want to restore a device (you already know the ECID via iTunes/Finder when connected). I want to keep this application simple and straightforward without overloading unnecessary features.

**Comment by lonkelle** [comment]
You are correct that it’s not an impossible to not get mixed up. But ECIDs aren’t human readable or memorable. Like, if a user has 5 devices, they'd have to stare at your profiles for a long time to figure out which one they didn't add because your profiles don't have names. Most objects in the digital world called "profiles" have custom names so the end user knows what they're looking at immediately.

I'm not saying there's a need for this, but do you at least understand the want for a human readable title for the profile now from a user's perspective?

**Comment by lucaIz-ldx** [comment]
I always keep all ECIDs for my devices in notes so that I simply do drag and drop to create profile. Anyway, I can't say this feature is useless now after you explained it. Since very few people know this application, it is not worth it for me to spend time to work on unimportant features. If you are willing to help me promote this project and more users are requesting for it, I might change my mind. For now, I will label it as an enhancement instead.
